### PR TITLE
Final push - ready for review

### PR DESCRIPTION
**Pull request by karldudley**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
+const express = require('express');
+const morgan = require('morgan');
+const mongoose = require('mongoose');
+const Telegraph = require('./models/telegraph');
+const { render } = require('ejs');
```
Comment (bradneve) [inline app.js:5]: EJS seems super helpful

File: views/details.ejs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    trashcan.addEventListener('click', (e) => {
+      const endpoint = `/blogs/${trashcan.dataset.doc}`;
+      fetch(endpoint, {
+        method: 'DELETE',
+      })
+      .then(response => response.json())
+      .then(data => window.location.href = data.redirect)
+      .catch(err => console.log(err));
+    });
+
```
Comment (bradneve) [inline views/details.ejs:28]: Anyone can delete any post... only tiny issue I can think of lol

**Review by bradneve — COMMENTED** [review]
Hi. So good. Only thing id say is how come you chose to use mongo over sql. Had no clue what was going on in most of it never seen all that ejs stuff, but it seems really handy.

**Comment by shaeebali** [comment]
Great project. Feedback as per slack call. 

P.s. Try and add in more frequent commits, otherwise excellent work!

**Comment by shaeebali** [comment]
Last thing I noticed for possible future improvement. Perhaps you could implement the MVC model.

File: app.js (hunk 1 above)
Comment (karldudley) [reply app.js:5]: Yeah, I really like it

File: views/details.ejs (hunk 2 above)
Comment (karldudley) [reply views/details.ejs:28]: Yeah, gonna try auth auth today to see if that will fix it

**Pull request merged**